src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -723,16 +723,10 @@ namespace webnn_native { namespace ie {
             return DAWN_INTERNAL_ERROR("Not support 'layout = rzn' now.");
         }
         const char* activations[2];
-        if (options->activations.resetGateActivation == nullptr) {
-            activations[0] = "sigmoid";
-        } else {
-            activations[0] = GetGruActivation(options->activations.resetGateActivation);
-        }
-        if (options->activations.newGateActivation == nullptr) {
-            activations[1] = "tanh";
-        } else {
-            activations[1] = GetGruActivation(options->activations.newGateActivation);
-        }
+        auto activationArray = gru->GetActivations().Get();
+        activations[0] = GetGruActivation(activationArray->Get(0));
```

Comment:
It's better to get all activations at once, 
std::vector<std::string> GetGruActivationNames(const Ref<OperatorArrayBase>& operators) {
}
std::vector<std::string> activations = GetGruActivationNames(gru->GetActivations());

then use it `activations.data()`